asserts/account_test.go

```diff
@@ -158,6 +158,7 @@ func (s *accountSuite) TestCheckUntrustedAuthority(c *C) {
 	otherDB := setup3rdPartySigning(c, "other", storeDB, db)
 
 	headers := ex.Headers()
+	delete(headers, "authority-id")
```

Comment:
ok, I did the comment thing in one but actually adjusted other tests differently and/or too be less non-sensical